test/cmdlineTests/exp_base_literal/output

```diff
@@ -1,13 +1,14 @@
 IR:
-/*******************************************************
- *                       WARNING                       *
- *  Solidity to Yul compilation is still EXPERIMENTAL  *
- *       It can result in LOSS OF FUNDS or worse       *
- *                !USE AT YOUR OWN RISK!               *
- *******************************************************/
+/*****************************************************|
+|                       WARNING                       |
+|  Solidity to Yul compilation is still EXPERIMENTAL  |
+|       It can result in LOSS OF FUNDS or worse       |
+|                !USE AT YOUR OWN RISK!               |
+|*****************************************************/
 
 
 object "C_81" {
+    /// @src 0:82,370
```

Comment:
Does this work here? Maybe we have to put it inside the `code{`

Reply:
I think it won't because the AsmParser won't read it otherwise.

Reply:
@christianparpart what won't work? Having it inside `code` or this?

Reply:
the `code {` part is not parsed by the AsmParser, so it's not under my control, is it?